aksel.nav.no/website/components/sanity-modules/token-table/ShowMore.tsx

```diff
@@ -0,0 +1,149 @@
+   */
+  scrollBackOnCollapse?: boolean;
+}
+
+/**
+ * A component for partially hiding less important content.
+ *
+ * @see [📝 Documentation](https://aksel.nav.no/komponenter/core/show-more)
+ * @see 🏷️ {@link ShowMoreProps}
+ *
+ * @example
+ * <ShowMore heading="Facts about toads">
+ *   Toads have dry, leathery skin, short legs, and large bumps covering the parotoid glands.
+ * </ShowMore>
+ */
+export const ShowMore = forwardRef<HTMLElement, ShowMoreProps>(
+  (
+    {
+      as: Component = "aside",
+      children,
+      size = "medium",
+      variant = "default",
+      collapsedHeight = "10rem",
+      /* heading,
+      headingSize = "medium",
+      headingLevel = "1", */
+      scrollBackOnCollapse = true,
+      className,
+      "aria-labelledby": ariaLabelledby,
+      ...rest
+    },
+    ref
+  ) => {
+    const localRef = useRef<HTMLElement>(null);
+    const mergedRef = useMemo(() => mergeRefs([localRef, ref]), [ref]);
+    const [isOpen, setIsOpen] = useState(false);
+    const ariaLabelId = useId();
+
+    const ChevronIcon = isOpen ? ChevronUpIcon : ChevronDownIcon;
+
+    return (
+      <Component
+        ref={mergedRef}
+        className={cl(
+          "navds-show-more",
+          `navds-show-more--${variant}`,
+          className,
+          { "navds-show-more--closed": !isOpen }
+        )}
+        aria-labelledby={
+          !ariaLabelledby && !rest["aria-label"] ? ariaLabelId : ariaLabelledby
+        }
+        {...rest}
+      >
+        {/* <Heading size={headingSize} level={headingLevel} id={ariaLabelId}>
+          {heading}
+        </Heading> */}
+
+        <div className="navds-show-more__button-section">
+          <div className="navds-show-more__button-wrapper">
+            <Button
+              type="button"
+              variant="secondary-neutral"
+              className="navds-show-more__button"
+              icon={<ChevronIcon aria-hidden />}
+              iconPosition="right"
+              size={size}
+              onClick={() => {
+                setIsOpen(!isOpen);
+                if (scrollBackOnCollapse && isOpen) {
+                  localRef.current?.scrollIntoView();
+                }
+              }}
+            >
+              {isOpen ? "Vis mindre" : "Vis mer"}
+            </Button>
+          </div>
+        </div>
+
+        <div
+          className="navds-show-more__content"
+          style={isOpen ? {} : { height: collapsedHeight }}
+          // @ts-expect-error https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60822
+          inert={isOpen ? undefined : ""}
+        >
+          {children}
+        </div>
+      </Component>
+    );
+  }
+);
+
+export default ShowMore;
+
+/**
+ * Erfaringer med bruk:
+ * - Skal heading være mandatory?
+ * - Bleed teknikk for å fjerne padding fungerer ikke internt med ShowMore siden `overflow: hidden;`. Må legges rundt ShowMore
+ *   Innholdet blir da "kuttet" når ShowMore er lukket (ser bra ut når den er åpen)
```

Comment:
Hva mener du? Ser fint ut for meg. Måtte du gjøre noe spesielt for å få det til å fungere?

Reply:
I de fleste tilfeller vil man bruke bleed internt i komponent, f.eks
```jsx
<div style="padding:4rem">
    <Bleed margin="4rem"/>
</div>
```
Men i Showmore må man gjøre: 
```jsx
<Bleed>
    <ShowMore >
       ...
    </ShowMore>
</Bleed>
```
Resultatet blir det samme, så har ikke så mye å si i dette tilfellet. Er ikke noe Showmore bør legge til rette for heller(og tror ikke det ville vært mulig 🤷 )

Reply:
Ah, skjønner 😅 Ser ikke for meg at det er mulig, nei. Ulempen med å ha bleed utenfor er at parent må ha en padding som er minst like stor som bleed sin negative marg.